docs/core/use-atom.mdx

```diff
@@ -64,26 +65,20 @@ function useAtom<Value>(
 ): [Value, never]
 ```
 
-The useAtom hook is to read an atom value stored in the Provider. It returns the atom value and an updating function as a tuple, just like useState. It takes an atom config created with `atom()`. Initially, there is no value stored in the Provider. The first time the atom is used via `useAtom`, it will add an initial value in the Provider. If the atom is a derived atom, the read function is executed to compute an initial value. When an atom is no longer used, meaning all the components using it are unmounted, and the atom config no longer exists, the value is removed from the Provider.
-
-```js
-const [value, setValue] = useAtom(anAtom)
-```
-
-The `setValue` takes one argument, which will be passed to the third argument of writeFunction of the atom. The behavior depends on how the writeFunction is implemented.
-
 ### How atom dependency works
+ 
+Every time we invoke the "read" function, we refresh the dependencies and dependents.
 
-To begin with, let's explain this. In the current implementation, every time we invoke the "read" function, we refresh the dependencies and dependents. For example, If A depends on B, it means that B is a dependency of A, and A is a dependent of B.
+> The read function is the first parameter of the atom.
+If B depends on A, it means that A is a dependency of B, and B is a dependent on A.
```

Comment:
I thought it made more sense to invert A and B, since B is derived from A, hence it "comes after" it